modules/dmrpp_module/DmrppArray.cc

```diff
@@ -1364,6 +1364,7 @@ void DmrppArray::read_linked_blocks(){
     for (unsigned int i = 1; i < num_linked_blocks; i++)
         accumulated_lengths[i] = individual_lengths[i-1] + accumulated_lengths[i-1];
 
+#if 0
```

Comment:
I think these #if 0 ... #endif blocks should be removed unless we are pretty sure they will be used later on.

Reply:
Remove it or leave a comment in the next pull request.